packages/camera/README.md

```diff
@@ -41,6 +41,10 @@ Change the minimum Android sdk version to 21 (or higher) in your `android/app/bu
 minSdkVersion 21
 ```
 
+Important to note is that the `MediaRecorder` class is not working properly on emulators. This makes
+that when a video is recorded with sound enabled, the duration won't be correct and you will only
+see the first frame.
```

Comment:
Can you also include the link to the official article which confirms recording video on the Android emulator is not working correctly?

Also a small suggestion on improving the text:
```suggestion
Important to note is that the `MediaRecorder` class is not working properly on emulators. Specifically when recording a video with sound enabled and try to play it back, the duration won't be correct and you will only
see the first frame.
```